test/tests/test_ingress.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+#test:disabled
```

Comment:
Would be good to add a small note that this is disabled only for Kind tests and for GKE/EKS tests should be enabled (Once we get that framework in place)

Reply:
Added note